### DISCUSSION TITLE
Einstein Sidebar not showing up in vs code

### DISCUSSION THREAD

**Discussion by rajnimtr** [forcedotcom/Einstein-GPT-for-Developers#62] [Q&A] [answered]
My Vs Code version  is 1.88.0. Einstein for Dev version is v0.43.1. The sidebar is not available to input the prompt. The status bar is showing ready.

**Comment by alan-liu-ofx** [comment]
I have the same issue after the recent update.

**Reply by rajnimtr** [reply]
I have few team members also reporting this issue hence opening this as an issue.

**Comment by rajnimtr** [comment, marked as answer]
This is fixed in the v0.43.2 version
